gamemodes/terrortown/gamemode/server/sv_player.lua

```diff
@@ -929,9 +783,12 @@ end
 -- @param Player|Entity attacker @{Player} or @{Entity} that killed the victim
 -- @hook
 -- @realm server
--- @ref https://wiki.garrysmod.com/page/GM/PlayerDeath
+-- @ref https://wiki.facepunch.com/gmod/GM:PlayerDeath
 -- @local
 function GM:PlayerDeath(victim, infl, attacker)
+	victim:SetLastDeathPosition(victim:GetPos())
+	victim:SetDiedInRound(true)
```

Comment:
If we adding a storing variable here, why not using a counter instead? This could be useful in the future too.

Reply:
nice idea, should this be reset on roundend? or on mapchange?

Reply:
On roundend i think, same as you doing currently. On mapchange would be useless, therefor you can use the `ply:Death()` var (ref.: https://wiki.facepunch.com/gmod/Player:Deaths)